org.eclipse.wb.swing/src/org/eclipse/wb/internal/swing/model/property/editor/color/CustomColorPickerComposite.java

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
+ * Copyright (c) DSA.

```

Comment:
Normally it has a first and last change year date. I'm not sure if it is required:

Copyright (c) 2021, 2021 DSA

Also, who is DSA? I could not find it on google. you have to uniquely identify the copyright holder. Maybe at a web address:

DSA Automobile Company (https://dsa.com)

Please also note that your company should agree that they put this in open source:

https://www.eclipse.org/legal/legalfaq.php#h.l3bvwrmty0iz

Reply:
Thank you for the feedback. :) I've spoken to my team lead at DSA about the public license and as far as is my understanding they have agreed to put that code in open source. I'll add their web address to the licenses and just ask my team lead to confirm with whomever he has to confirm with before making another commit.

When you say "first and last change year date" you're referring to the code changes right?

Reply:
Yes, changes in that source.

Reply:
Okay thank you. Then I believe the changes I made should be right? Sorry for all the confusion. Im new to WB and git

Reply:
No problem, Marcel. Thank you for contributing. You can ask me anything. Please ping me if I am not quick enough to respond ('pinging': reply on the issue with "ping".)